sdk/core/Azure.Core.TestFramework/src/RecordedTestBase.cs

```diff
@@ -154,6 +154,12 @@ public string ReplacementHost
         {
         };
 
+        /// <summary>
+        /// Whether or not to enable the test framework to proxy traffic through fiddler. This should only be set to
+        /// True for local debugging scenarios, when you have Fiddler listening for traffic.
+        /// </summary>
+        public bool EnableFiddler { get; set; }
```

Comment:
Can we read it from the environment also? That would make it less likely to check in.

Reply:
I'd rather not add the overhead/edge cases that comes along with trying to detect this automatically. There is no danger if someone checks it in - it is similar to accidentally checking in RecordedTestMode.Record - the CI will fail.

Reply:
I think we should avoid popular config options that are easy to check in and ruin a CI run.

Reply:
I don't think this will be a popular option.

Reply:
It will be for me 😄

Reply:
Oh you mean read it from like a flag that someone sets in the env?

Reply:
Ok I think that is reasonable. @seanmcc-msft any concerns if we implement this as an env var that you would set?

Reply:
No that is fine.